### PR TITLE
increase test timeout, use local manifest file

### PR DESCRIPTION
**Pull request by rkpagadala**
increase test timeout to 22h instead of 1 hour
use local (previously copied) manifest file for publish_release script

### CODE REVIEW

File: release/airflow/cloud_tag.template.json [hunk 1]
```diff
@@ -9,10 +9,22 @@
       "args": [ "chmod", "u+x", "./istio_checkout_code.sh" ]
     },
     {
+      "volumes": [
+	{
+          "name": "buildoutput",
+          "path": "/output"
+	}
+      ],
```
Comment (hklai) [inline release/airflow/cloud_tag.template.json]: where did output go before?

File: release/airflow/cloud_test.template.json [hunk 2]
```diff
@@ -16,5 +16,5 @@
   "options": {
     "machineType": "N1_HIGHCPU_8"
   },
-  "timeout": "3600s",
+  "timeout": "22h",
```
Comment (hklai) [inline release/airflow/cloud_test.template.json]: Prob should be higher than the actual test timeout.

File: release/airflow/cloud_tag.template.json (hunk 1 above)
Comment (rkpagadala) [reply release/airflow/cloud_tag.template.json]: it was copied only when /output is present. some steps did not need it

**Pull request merged**